hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerDoubleBuffer.java

```diff
@@ -510,7 +556,8 @@ public long getFlushIterations() {
   public synchronized CompletableFuture<Void> add(OMClientResponse response,
       long transactionIndex) {
     currentBuffer.add(new DoubleBufferEntry<>(transactionIndex, response));
-    notify();
+    if (! (response instanceof OMKeyCommitResponse))
```

Comment:
This was done for testing similar to https://github.com/GeorgeJahad/ozone/compare/05ed641c6...6cdfd890f#

Revert it back in next revision.